.gitignore

```diff
@@ -23,6 +23,7 @@ dist/
 docs/_build/
 treeherder/media/
 treeherder/static/
+treeherder/webapp/log_cache/
```

Comment:
This entry was removed deliberately in bug 1328469 - please don't re-add it :-)
If your git status is showing files locally they need to be deleted.